### PR TITLE
Clean up logging

### PR DESCRIPTION
**Pull request by lrettig**
## Motivation

A developer should be able to tell what's going on by following the node log. This cleans up a lot of the noise in the logs and highlights important information such as sync status. No log lines are removed, but verbosity is decreased (e.g., warn -> info, info -> debug). libp2p logs in particular are moved to warn level.

## Test Plan

N/A, no logic changes

## TODO

- [x] Explain motivation or link existing issue(s)
- [x] Test changes and document test plan
- [ ] Update documentation as needed
- [ ] Update [changelog](../CHANGELOG.md) as needed

### CODE REVIEW

File: fetch/mesh_data.go [hunk 1]
```diff
@@ -110,6 +112,7 @@ func (f *Fetch) getHashes(
 	}
 
 	eg.Wait()
+	f.logger.WithContext(ctx).With().Info("finished fetching hash objects")
```
Comment (pigmej) [inline fetch/mesh_data.go:115]: IMO this it debug

File: fetch/mesh_data.go [hunk 2]
```diff
@@ -68,6 +68,8 @@ func (f *Fetch) getHashes(
 
 	var eg errgroup.Group
 	var failed atomic.Uint64
+	f.logger.WithContext(ctx).With().Info("attempting to fetch hash objects",
```
Comment (pigmej) [inline fetch/mesh_data.go:71]: IMO debug too.

File: miner/proposal_builder.go [hunk 3]
```diff
@@ -387,7 +387,7 @@ func (pb *ProposalBuilder) decideMeshHash(ctx context.Context, current types.Lay
 }
 
 func (pb *ProposalBuilder) UpdateActiveSet(epoch types.EpochID, activeSet []types.ATXID) {
-	pb.logger.With().Info("received activeset update",
+	pb.logger.With().Debug("received activeset update",
```
Comment (pigmej) [inline miner/proposal_builder.go:390]: That one IMO is Info because it's the centralized update.

File: node/node.go [hunk 4]
```diff
@@ -1856,8 +1856,10 @@ func (app *App) startSynchronous(ctx context.Context) (err error) {
 	cfg := app.Config.P2P
 	cfg.DataDir = filepath.Join(app.Config.DataDir(), "p2p")
 	p2plog := app.addLogger(P2PLogger, lg)
-	// if addLogger won't add a level we will use a default 0 (info).
-	cfg.LogLevel = app.getLevel(P2PLogger)
+
+	// our p2p code logs at the default level (Info), but we want the lower level libp2p logging
+	// to be less verbose
+	cfg.LogLevel = zapcore.WarnLevel
```
Comment (pigmej) [inline node/node.go:1862]: it's configurable; why not change the default config instead? Sometimes we really need to set it.

File: p2p/upgrade.go [hunk 5]
```diff
@@ -409,7 +409,7 @@ func (fh *Host) trackNetEvents() error {
 				return nil
 			}
 			natEv := ev.(event.EvtNATDeviceTypeChanged)
-			fh.logger.With().Info("NAT type changed",
+			fh.logger.With().Debug("NAT type changed",
```
Comment (pigmej) [inline p2p/upgrade.go:412]: Will be hard to detect what's wrong when we hide it with Debug.

File: syncer/data_fetch.go [hunk 6]
```diff
@@ -326,6 +326,12 @@ func (d *DataFetch) GetEpochATXs(ctx context.Context, epoch types.EpochID) error
 		log.Int("missing", len(missing)),
 	)
 	if len(missing) > 0 {
+		d.logger.WithContext(ctx).With().Info("fetching missing epoch atxs",
+			epoch,
+			log.Stringer("peer", peer),
+			log.Int("total", len(ed.AtxIDs)),
```
Comment (pigmej) [inline syncer/data_fetch.go:332]: afaik, this total is not the absolute total, so we need to be careful here.

maybe @dshulyak can point to more, but I remember that I had a similar idea some time ago, and that's how I remembered.

Comment (dshulyak) [reply syncer/data_fetch.go:332]: i think this is from epoch info response, so in this case this is how many atxs are available from that peer. definitely not absolute, and not even how many are available locally 

> ed, err := d.fetcher.PeerEpochInfo(ctx, peer, epoch)

**Comment by lrettig** [comment]
this is stale by now, closing and will revisit these later if needed.

**Pull request closed without merging**